.vitepress/theme/components/VueJobs.vue

```diff
@@ -1,14 +1,29 @@
 <script lang="ts">
 // shared data across instances so we load only once
 const base = `https://app.vuejobs.com/feed/vuejs/docs?format=json`
-let items = $ref([])
+
 </script>
 
 <script setup lang="ts">
 import { onMounted, computed } from 'vue'
 
 let vuejobs = $ref<HTMLElement>()
 
+let items = $ref<Jobs[]>([])
+
+type Jobs = {
+  organization: Organization;
+  title: string;
+  link: string;
+  locations: Array<string>;
+  remote: false | "ALLOWED" | "ONLY";
```

Comment:
I suggest using `string[]` instead of `Array<string>`, and single quotes for strings. I think that'd be more consistent with the usual code style used in the docs.

```suggestion
  locations: string[];
  remote: false | 'ALLOWED' | 'ONLY';
```